gtest/src/manager.rs

```diff
@@ -446,6 +449,36 @@ impl ExtManager {
 
         core_processor::handle_journal(journal, self);
     }
+
+    /// Call non-void meta function from an actor stored in manager.
+    /// Warning! This is a static call that doesn't change actors pages data.
+    pub(crate) fn call_meta(
+        &mut self,
+        program_id: &ProgramId,
+        payload: Payload,
```

Comment:
here option also

Reply:
for `Program` might be call_meta_empty() and call_meta(arg) in both variants bytes and encodable